app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div>
+  <h1>LIST OF ORGANISATIONS</h1>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Address</th>
+        <th>Email</th>
+        <th>Contact No.</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @organisations.each do |organisation| %>
+        <tr>
+          <td><%= organisation.name %></td>
+          <td><%= organisation.address %></td>
+          <td><%= organisation.email %></td>
+          <td><%= organisation.contactNo %></td>
```

Comment:
This attribute name is a bit non-standard for Rails - I would expect it to be `contact_number`